bundles/org.openhab.binding.keba/src/main/resources/ESH-INF/thing/kecontact.xml

```diff
@@ -37,6 +37,9 @@
 			<channel id="display" typeId="display" />
 			<channel id="authreq" typeId="switch" />
 			<channel id="authon" typeId="switch" />
+            <channel id="sessionRFIDtag" typeId="sessionRFIDtag" />
```

Comment:
Please note that it seems you mixed up two types of indentation, the default for our XML files (non POM) is tabs, please make it consistent.